transformations/yoda_transform/README.md

```diff
@@ -0,0 +1,38 @@
+# Yoda Transformation
+This perturbation modifies sentences to flip the clauses such that it reads like "Yoda Speak". For example,
+"Much to learn, you still have". This form of construction is sometimes called "XSV", where "the “X” being a stand-in
+for whatever chunk of the sentence goes with the verb", and appears very rarely in English normally. For a more in-depth
+description of "Yoda Speak" linguistically,
+see, for example, [this article](https://www.theatlantic.com/entertainment/archive/2015/12/hmmmmm/420798/). The rarity of
+this construction in ordinary language makes it particularly well suited for NL augmentation and serves as a relatively easy
+but potentially powerful test of robustness.
+
+Author names: Ryan Teehan, Vinay Prabhu, Nick Roberts, Sang Han
+
+Author emails: rsteehan@gmail.com, vinay@unify.id, nick11roberts@cs.wisc.edu, sanghan@protonmail.com
+
+Author Affiliations: Charles River Analytics, UnifyID, UW Madison, Independent Researcher
+
+## What type of a transformation is this?
+This transformation acts like a perturbation to test robustness. Sentences with clauses transformed in this way are
+comprehensible by English speakers, but may pose serious difficulty to NL algorithms. In particular, it rearranges the components
+of an English sentence such that the verb comes at the end. This is rare in normal English speech but does not pose much
+difficulty for English speakers.
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification,
+text generation, etc, but may be especially useful for robustness of POS tagging algorithms. Modifying sentence structure
+in this way may expose an algorithm's reliance on spurious correlations in the training data.
+## Previous Work
+My implementation of Yoda Speak borrowed from this code: https://github.com/yevbar/Yoda-Script
```

Comment:
"My"? I thought this transformation had several authors

Reply:
We all collaboratively wrote a few different transformations, but when I wrote the README for this one I defaulted to the singular because it seemed more natural. I'll correct that